lib/mixins/index.js

```diff
@@ -1,11 +1,21 @@
-import clickOutMixin from './clickout';
-import dropdownMixin from './dropdown';
-import formMixin from './form';
-import formCheckBoxMixin from './form-checkbox';
-import formCustomMixin from './form-custom';
-import formOptionsMixin from './form-options';
-import linkMixin from './link';
-import listenOnRootMixin from './listen-on-root';
-import popoverMixin from './popover';
+import clickOutMixin from "./clickout"
```

Comment:
My darn editor tools! Like I said before, if I can configure the tool to format for me, I'm good. I guess prettier orders double quotes.